src/util/rangeutil.hpp

```diff
@@ -194,3 +195,15 @@ Value max_value(const Seq& seq, Compare cmp = Compare{}) {
 } // namespace util
 } // namespace mc
 } // namespace nest
+
+// convenience helper for printing a range
+
+template <typename T>
+std::ostream& operator<<(std::ostream& o, const nest::mc::util::range<T>& rng) {
+    o << "{";
+    for (auto const& value: rng) {
+        o << value << " ";
+    }
+    return o << "}";
+}
```

Comment:
I'd prefer to have a pretty printer wrapper for this sort of thing, as the one formatter for all ranges is a bit of a Procrustean bed. Use would be nearly as simple though, e.g.
```
    std::cout << range_printer(some_range) << ...
```